main/2021/CG-05-25.md

```diff
@@ -25,6 +25,7 @@ Installation is required, see the calendar invite.
 1. Adoption of the agenda
 1. Proposals and discussions
     1. Review of action items from prior meeting.
+    1. Present [WebAssembly, UTF-8 and Web Platform APIs - Or: Please don't break the Web 🥺](https://youtu.be/IhvwO0zkbXU) (Daniel Wirtz) [15 mins]
```

Comment:
I would like this title to be changed. None of the proposals/features brought up [here](https://github.com/WebAssembly/design/issues/1407) "break the Web" - this is an absolute principle we must not violate in our Web-facing semantics. Characterising the discussion in this way is not a productive first step.

I agree that there is legitimate debate to be had about performance and ecosystem fragmentation, which I would welcome so long as we can avoid hyperbole.

Reply:
In the presentation, I mean and hope to manage to express it as a friendly wink lessening the tension from foregoing heated discussions (I read the "pleading face" for instance), while at the same time hinting at the problem the presentation will discuss, as it is indeed pointing out where the Web's design principle of backwards compatibility may be violated.

But thank you for the heads up! I will remove the subtitle from this PR however, as I agree that, without context, it is easily interpreted this way.